packages/agents/src/lib/sseEdge.ts

```diff
@@ -139,3 +144,32 @@ export class SSEEdgeTransport implements Transport {
     this.controller.enqueue(new TextEncoder().encode(messageText));
   }
 }
+
+export class SSEEdgeClientTransport extends SSEClientTransport {
+  /**
+   * Creates a new EdgeSSEClientTransport, which overrides fetch to be compatible with the CF workers environment
+   */
+  constructor(
+    private url: URL,
+    options: SSEClientTransportOptions
+  ) {
+    const fetchOverride = (url: any, options = {}) => {
+      const workerOptions = {
+        ...options,
+      };
+      // Remove unsupported properties
+      // @ts-ignore
+      delete workerOptions.mode;
```

Comment:
We have to do this because of lack of support for request.mode property in runtime?

https://developer.mozilla.org/en-US/docs/Web/API/Request/mode

Reply:
yup